starters/basic-starter/tailwind.config.js

```diff
@@ -13,6 +15,11 @@ module.exports = {
           lg: '940px',
         },
       },
+      colors: {
+        accent: colors.blue['600'],
+        secondary: colors.gray['600'],
+        body: colors.gray['500'],
```

Comment:
Body seems to be an odd description for this color. Maybe we could just call it gray?

Reply:
'text-color-gray' as a className was giving me an issue so that is why the new alias name is 'grey' instead of 'gray' for now.